### ISSUE TITLE
Research Dashboard Customization

### ISSUE THREAD

**Issue by lukas-rotsching** [amosproj/amos2023ss02-open-search-meta-data-hub#14]
## User story
1. As a user
2. I want to customize the look of the dashboard (e.g. change the title, logo, color scheme...)
3. So that i can adapt the dashboard to my own company's design

## Acceptance criteria
* Research what aspects of the dashboard can be customized and how.
* Evaluate for each aspect how hard it would be to actually implement the change. What would need to be done?
* Evaluate if it would be possible to create simple config file or tool that gives customers an easy way to customize the looks of the dashboard. Evaluate which customizations could be done in such a way.
* Write down the results of your research!

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
